ext/hash/php_hash_blake3.h

```diff
@@ -0,0 +1,35 @@
+#ifndef PHP_HASH_BLAKE3_H
+#define PHP_HASH_BLAKE3_H
+
+#include "ext/standard/basic_functions.h"
+#include "php_hash.h"
+#include "blake3/upstream_blake3/c/blake3.h"
+
+
+// typedef struct blake3_hasher PHP_BLAKE3_CTX;
+#define PHP_BLAKE3_CTX blake3_hasher
+// help: is V correct?
+//#define PHP_BLAKE3_SPEC "b8b8qb64bbbbb1760"
+#define PHP_BLAKE3_SPEC "L8L8Qa64CCCCL8Ca1760"
```

Comment:
can someone double-check this? because i couldn't wrap my head around it, and this is just my best guess